### DISCUSSION TITLE
How to package entry_points.txt in the bundle

### DISCUSSION THREAD

**Discussion by jshen28** [pyinstaller/pyinstaller#8875] [Help] [answered]
Hello,

Thank you very much for the great work. I am a newbie to pyinstaller and would like to get help on packaging a python project using OpenStack python clients. 

The particular problem I am facing is the bundled executable could not find **entry_points.txt** file which saves python module information which is used to be loaded at runtime.

Looking forward for the help. Thank you very much.

Best Regards,
Norman Shen

**Comment by jshen28** [comment]
I found option `--copy-metadata` might do the trick?

**Reply by bwoodsend** [reply, marked as answer]
Yep, that's the one.

**Comment by jshen28** [comment]
Thank you for help.
